packages/nextjs/src/server/authMiddleware.test.ts

```diff
@@ -449,19 +436,12 @@ describe('Dev Browser JWT when redirecting to cross origin', function () {
   });
 
   it('does NOT append the Dev Browser JWT if x-clerk-redirect-to header is not set', async () => {
-    mockRedirectToSignIn.mockReturnValueOnce(
-      NextResponse.redirect(
-        'https://accounts.included.katydid-92.lcl.dev/sign-in?redirect_url=https%3A%2F%2Fwww.clerk.com%2Fprotected',
-      ),
-    );
     const resp = await authMiddleware({
-      beforeAuth: () => NextResponse.next(),
+      beforeAuth: () => NextResponse.redirect('https://google.com/'),
     })(mockRequest({ url: '/protected', appendDevBrowserCookie: true }), {} as NextFetchEvent);
 
     expect(resp?.status).toEqual(307);
-    expect(resp?.headers.get('location')).toEqual(
-      'https://accounts.included.katydid-92.lcl.dev/sign-in?redirect_url=https%3A%2F%2Fwww.clerk.com%2Fprotected',
-    );
+    expect(resp?.headers.get('location')).toEqual('https://google.com/');
```

Comment:
🙃 Could you also assert that the `x-clerk-redirect-to` is not set? This is not a change related to this PR but it will improve the test to match the description.